data/json/items/ammo/shot.json

```diff
@@ -84,7 +84,7 @@
     "copy-from": "shot_flechette",
     "type": "AMMO",
     "name": { "str": "flechette shell, black powder", "str_pl": "flechette shells, black powder" },
-    "description": "A shotgun shell filled with tiny steel darts.  Extremely damaging, plus the spread makes it very accurate at short range.  Slices through most forms of armor with ease.  Someone was down on their luck when they hand-reloaded this one - it's filled with black powder instead of smokeless powder.  Expect lower velocity, muzzle smoke, and a dirtier barrel if you shoot it.",
+    "description": "A shotgun shell filled with tiny steel darts.  The spread makes it very accurate at short range.  Someone was down on their luck when they hand-reloaded this one - it's filled with black powder instead of smokeless powder.  Expect lower velocity, muzzle smoke, and a dirtier barrel if you shoot it.",
```

Comment:
```suggestion
    "description": "A shotgun shell filled with tiny steel darts.  Despite the fins on the darts, insufficient stabilization means most will not hit targets point forward, leading to poor penetration and huge patterning.  A gimmick at best.  Someone was down on their luck when they hand-reloaded this one - it's filled with black powder instead of smokeless powder.  Expect lower velocity, muzzle smoke, and a dirtier barrel if you shoot it.",
```